src/ducks/transactions/scroll/InfiniteScroll.jsx

```diff
@@ -24,7 +24,7 @@ class InfiniteScroll extends React.Component {
   componentDidMount() {
     this.scrollingElement = this.getScrollingElement()
     this.listenToScroll()
-    this.checkForLimits()
+    setTimeout(() => this.checkForLimits(), 0)
```

Comment:
Maybe add a comment to explain why we use a `setTimeout` here? I'm afraid in a few months we don't remember why.

Reply:
Yes, I added a comment, thanks !

Also, I went with another approach following crash's comment. Since checkForLimits is invoked in cDU *and* cDM, I changed the throttle options to deactivate the *leading* call, which prevents the call to happen during cDU *and* cDM.